mux.c

```diff
@@ -2268,7 +2268,13 @@ muxclient(const char *path)
 		fatal("ControlPath too long ('%s' >= %u bytes)", path,
 		    (unsigned int)sizeof(addr.sun_path));
 
-	if ((sock = socket(PF_UNIX, SOCK_STREAM, 0)) == -1)
+	#ifdef HAVE_AFUNIX_H
+	sock = w32_afunix_socket(&addr);
+	#elif
```

Comment:
Is this elif intentional? It doesn't even compile.

Reply:
@thedarkcolour That's wrong, it should be an `#else`. I'll fix it.